src/Tribe/REST/V1/Endpoints/Archive_Event.php

```diff
@@ -73,6 +73,13 @@ public function get( WP_REST_Request $request ) {
 
 		$args['post_parent'] = $request['post_parent'];
 
+
+		// Allow REST API users to define the default behavior of the inclusive date parameters.
+		$use_inclusive_start_end_dates = true;
+		if ( isset( $request['strict_dates'] ) ) {
+			$use_inclusive_start_end_dates = ! $request['strict_dates'];
+		}
```

Comment:
Suggestion: ternary.
```suggestion
		$use_inclusive_start_end_dates = isset( $request['strict_dates'] ) ? ! $request['strict_dates'] : true;
```